### PR TITLE
Cms 1246 accessibility bugs

### PR DESCRIPTION
**Pull request by Kenneth-Siu**

### CODE REVIEW

File: contact/templates/contact/routing/step-location.html [hunk 1]
```diff
@@ -12,6 +12,6 @@ <h2 class="heading-large">Where are you based?</h2>
 
 {% block back_button %}
     {% if prev_url %}
-        <a href="{{ prev_url }}" class="previous-step">Back</button>
+        <a href="{{ prev_url }}" class="previous-step">Back</a>
```
Comment (kowalcj0) [inline contact/templates/contact/routing/step-location.html]: :+1:

File: article/templates/article/components/accordions.html [hunk 2]
```diff
@@ -30,7 +30,7 @@
                 <div class="column-full">
                 {% endif %}
                   {% if subsection.icon %}
-                  <img class="accordion-icon" src="{{ subsection.icon.url }}"/>
+                  <img class="accordion-icon" src="{{ subsection.icon.url }}" alt="{{ subsection.heading }} icon"/>
```
Comment (kowalcj0) [inline article/templates/article/components/accordions.html]: :+1:

File: core/static/js/dit.js [hunk 3]
```diff
@@ -86,154 +88,8 @@ dit.utils = (new function () {
   }
 
 });
-// Responsive Functionality.
```
Comment (kowalcj0) [inline core/static/js/dit.js]: @Kenneth-Siu 
A lot of (potentially useful) code was removed here. Will it affect responsiveness of our site in any way?

Comment (Kenneth-Siu) [reply core/static/js/dit.js]: This code is a duplicate of ones that are

a) already in directory-components, and
b) only used for the headers (and therefore should only live in directory-components)

No responsiveness functionality has been removed. 🌞

**Pull request merged**